syntax = "proto3";

package AcFunDanmu.Im.Cloud.Message;

import "SyncCookie.proto";

message GroupMemberListGetRequest {
	string groupId = 1;
	Im.Basic.SyncCookie syncCookie = 2;
}
